bayesian/utils.py

```diff
@@ -329,14 +329,23 @@ def search_packages_from_graph(tokens):
     return {'result': pkg_list}
 
 
-def get_analyses_from_graph(ecosystem, package, version):
+def get_analyses_from_graph(ecosystem, package, version, vendor=None):
     """Read analysis for given package+version from the graph database."""
     script1 = """\
-g.V().has('pecosystem', ecosystem).has('pname', name).has('version', version).as('version')\
-.in('has_version').dedup().as('package').select('version').coalesce(out('has_cve')\
-.as('cve').select('package','version','cve').by(valueMap()),select('package','version')\
-.by(valueMap()));\
-"""
+        g.V().has('pecosystem', ecosystem).has('pname', name).has('version', version).as('version')\
```

Comment:
Instead of using if else, use a separate function so that later if we need to add any filter etc, we can do it in 1 function

Reply:
Added